openhands/core/config/app_config.py

```diff
@@ -14,6 +16,21 @@
 from openhands.core.config.security_config import SecurityConfig
 
 
+def _get_or_create_jwt_secret() -> str:
+    config_dir = os.path.dirname(os.path.abspath(__file__))
+    secret_file = os.path.join(config_dir, '.jwt_secret')
+    
+    if os.path.exists(secret_file):
+        with open(secret_file, 'r') as f:
+            return f.read().strip()
+    
+    new_secret = uuid.uuid4().hex
+    os.makedirs(os.path.dirname(secret_file), exist_ok=True)
+    with open(secret_file, 'w') as f:
+        f.write(new_secret)
```

Comment:
When we run evals, we do it with multiprocessing, 16-32 processes, could we make it a little safer for that?

Also, even if we have it set, this would be executed. Maybe we can do it at the end, e.g. maybe [here](https://github.com/All-Hands-AI/OpenHands/blob/b11e90598860230b8c8560eabb643feb00362e25/openhands/core/config/utils.py#L211)?